libparsec/src/config.rs

```diff
@@ -47,6 +54,14 @@ impl From<ClientConfig> for libparsec_client::ClientConfig {
             workspace_storage_cache_size: config.workspace_storage_cache_size,
             proxy: ProxyConfig::default(),
             with_monitors: config.with_monitors,
+            prevent_sync_pattern: match config.prevent_sync_pattern {
+                Some(pattern) => Regex::from_glob_reader(
+                    "client_prevent_sync_pattern",
+                    std::io::Cursor::new(pattern),
+                )
+                .expect("Cannot process provided prevent sync pattern file"),
```

Comment:
`Regex` type should be exposed at bindings level, so providing an invalid `prevent_sync_pattern` cause a conversion error that javascript can deal with instead of having a panic)

See for instance `ParsecAddr` on how to implement the conversion code:
https://github.com/Scille/parsec-cloud/blob/9926f07483e6a7777a5aa87528d3a1957a282b80/bindings/generator/api/addr.py#L19-L23

Reply:
Will be done in another PR, an issue was created: https://github.com/Scille/parsec-cloud/issues/7923